python/cucim/src/cucim/skimage/_vendored/_ndimage_util.py

```diff
@@ -168,7 +168,7 @@ def _generate_boundary_condition_ops(mode, ix, xsize, int_t="int",
             {ix} %= {xsize};
             '''
             ops_lower = ops_upper + f'''
-            if ({ix} < 0) {{
```

Comment:
It doesn't seem like this fix should be necessary? I would think the modulus on line 168 makes it so that the xsize should not need to be added more than once.

Reply:
Asked about this in comment ( https://github.com/cupy/cupy/pull/7235#discussion_r1093835536 )